Formula/crane.rb

```diff
@@ -0,0 +1,22 @@
+class Crane < Formula
+  desc "Tool for interacting with remote images and registries"
+  homepage "https://github.com/google/go-containerregistry"
+  url "https://github.com/google/go-containerregistry/archive/v0.4.0.tar.gz"
+  sha256 "90ddaaf4a6b76c490e0664a55946342055ba70c0ed40d52724440fa8ac0b45db"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "-ldflags",
+      "-s -w -X github.com/google/go-containerregistry/cmd/crane/cmd.Version=#{version}",
+      "./cmd/crane"
+  end
+
+  test do
+    json_output = shell_output("#{bin}/crane manifest homebrew/brew")
```

Comment:
`crane manifest gcr.io/google_containers/pause` ?

Reply:
Would rather use `google-containers` -- the underscore thing is a historical curiosity.

Though, really, we should probably use something under the control of the project, e.g. `gcr.io/go-containerregistry/crane`?

Reply:
Thanks for looking at the test. Does either of you mind opening a PR for it?

Reply:
I'll send a PR.